src/renderer/app-paths/get-value-from-registered-channel/register-ipc-channel-listener.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import ipcRendererInjectable from "./ipc-renderer/ipc-renderer.injectable";
+import type {
+  IpcChannelListener,
+} from "../../ipc-channel-listeners/ipc-channel-listener-injection-token";
+
+const registerIpcChannelListenerInjectable = getInjectable({
+  id: "register-ipc-channel-listener",
+
+  instantiate: (di) => {
+    const ipc = di.inject(ipcRendererInjectable);
+
+    return ({ channel, handle }: IpcChannelListener) => {
+      ipc.on(channel.name, (_, data) => {
+        handle(data);
```

Comment:
I would call this `listener` instead because `handle` (and `handler`) implies waiting (an `ipc.invoke(...)`) instead of just an event handler (an `ipc.send(...)`)